src/pages/Login/index.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from "react";
+import Button from "../../components/Button/index";
```

Comment:
Holi! No hace falta importar el index! 

```suggestion
import Button from "../../components/Button";
```
```suggestion
import Button from "../../components/Button/index";
```

Reply:
Se cambio como solicitado